source/tyr/tyr/default_settings.py

```diff
@@ -112,6 +112,11 @@
         'schedule': timedelta(seconds=30),
         'options': {'expires': 25}
     },
+    'purge-autocomplete-every-30-seconds': {
```

Comment:
Every ten minutes? that fast, one time per day should be fine.
http://docs.celeryproject.org/en/latest/userguide/periodic-tasks.html#crontab-schedules